src/index.ts

```diff
@@ -423,6 +423,13 @@ export function create (rawOptions: CreateOptions = {}): Register {
 
   function reportTSError (configDiagnosticList: _ts.Diagnostic[]) {
     const error = createTSError(configDiagnosticList)
+
+    const stackLines = error.stack.split('\n')
+    const lastLine = stackLines.findIndex(line => line.includes('ts-node'))
+    if (lastLine !== -1) {
+      error.stack = stackLines.slice(0, lastLine).join('\n')
```

Comment:
Can't an error go across files (e.g. have encountered `ts-node` multiple times)?

Reply:
As it works right now this code drops the entire stack trace, since `ts-node` always has the last frame on the callstack, so the only case where this hack would work out poorly is if "ts-node" was present in the TypeScript error, which I could totally see occurring.

This was more of a "good enough" solution for what I needed, I wouldn't want to pretend this is foolproof.